setup.py

```diff
@@ -67,6 +67,7 @@ def enable_log_to_stdout(logname):
         str(plugin_name),  # https://github.com/pypa/setuptools/pull/597
     ],
     package_dir={plugin_name: plugin_name},
+    install_requires=["smpplib2==0.1.1"],
```

Comment:
That's fine, but if this is a library then we should make it more lenient (`>=0.1.1`). If we go by a hard equals (`==`) it may lead to unresolveable conflicts in the future.

It won't be a big problem with `smpplib2`, but let's set an example anyway for future developers for this library.